src/app/parts-editor/components/dialog-database/dialog-database.component.html

```diff
@@ -0,0 +1,20 @@
+<h1 i18n>Parts missing from database</h1>
+
+<form [formGroup]="missingPartsForm" (ngSubmit)="onSubmit()">
+  <div formArrayName="updatedParts">
+    <div *ngFor="let part of updatedParts().controls; let partIndex = index">
+      <div [formGroupName]="partIndex">
+        <input type="text" readonly formControlName="rowIndex" style="width: auto" />
```

Comment:
Never use inline styles. Also width auto is its default value so this does nothing.